src/ray/object_manager/pull_manager.cc

```diff
@@ -583,33 +668,101 @@ bool PullManager::PullRequestActiveOrWaitingForMetadata(uint64_t request_id) con
   return bundle_it->second.num_object_sizes_missing > 0;
 }
 
-std::string PullManager::BundleInfo(const Queue &bundles) const {
+std::string PullManager::BundleInfo(const Queue &bundles,
+                                    uint64_t highest_id_being_pulled) const {
   auto it = bundles.begin();
   if (it == bundles.end()) {
     return "N/A";
   }
   auto bundle = it->second;
   std::stringstream result;
   result << bundle.num_bytes_needed << " bytes, " << bundle.objects.size() << " objects";
+  if (highest_id_being_pulled) {
+    result << " (active)";
+  } else {
+    if (bundle.num_object_sizes_missing > 0) {
+      result << " (inactive, waiting for object sizes)";
+    } else {
+      result << " (inactive)";
+    }
+  }
   return result.str();
 }
 
+int64_t PullManager::NextRequestBundleSize(const Queue &bundles,
```

Comment:
Any way we can dedupe this with ActivateNextPullBundleRequest?

Reply:
I tried, it turned out difficult to read though :/ I think it's better to leave it slightly duplicated.